DigitalLearningSolutions.Data/DataServices/UserDataService/UserCentreDetailsDataService.cs

```diff
@@ -127,6 +127,28 @@ ELSE 0
             );
         }
 
+        public bool PrimaryEmailIsInUse(string email, IDbTransaction? transaction = null)
+        {
+            return connection.QueryFirst<bool>(
+                @"SELECT COUNT(*)

```

Comment:
I'm not a big fan of the implicit "Count can't be greater that 1 so we can treat the return result as a bool" because it's a bit... implicit. Can we query out an int instead, then return `count > 0`? Similar in other function.

Reply:
Done